src/analysis/retail/mage/frost/Guide.tsx

```diff
@@ -50,19 +51,22 @@ export default function Guide({ modules, events, info }: GuideProps<typeof Comba
   return (
     <>
       <Section title="Core">
+        <HideExplanationsToggle id="hide-explanations-cooldowns" />
```

Comment:
these should technically have different ids since this is actually a DOM id. browsers let it slide but generally should be distinct.

Reply:
they haven't changed much, @Sharrq already added Winter's Chill and Permafrost Lances, the only two new talents, on the Fire and Arcane update.

Reply:
ids changed